easydiffraction/Job.py

```diff
@@ -893,11 +890,14 @@ def print_free_parameters(self):
                 parameters['names'].append(parameter.display_name)
                 parameters['values'].append(parameter.raw_value)
                 parameters['errors'].append(parameter.error)
-                parameters['units'].append(f'{parameter.unit:~H}')
+                parameters['units'].append(f'{parameter.unit:~P}')
```

Comment:
at some point, we need to refactor out the plotting and printing methods from `Job.py`. This should be easily possible. (Not now, let's wait until you are done with the main functionality)

Reply:
Completely agree with that.